modules/google_service_account/inputs.tf

```diff
@@ -1,6 +1,10 @@
 variable "account_id" {
   description = "The account id that is used to generate the service account email address and a stable unique id. It is unique within a project, must be 6-30 characters long, and match the regular expression [a-z]([-a-z0-9]*[a-z0-9]) to comply with RFC1035. Changing this forces a new service account to be created."
   type        = string
+  validation {
+    condition     = length(var.account_id) >= 6 && length(var.account_id) <= 30 && regex("[a-z]([-a-z0-9]*[a-z0-9])", var.account_id) == var.account_id
```

Comment:
[Error message](https://app.terraform.io/app/honestbank/workspaces/iac-iam-test/runs/run-BPAThMmyC9rBSZFP):
```
Error: "account_id" ("terraform-ml-infrastructure-test-bdf8d5a5") doesn't match regexp "^[a-z](?:[-a-z0-9]{4,28}[a-z0-9])$"
```
Should we use the same regex?

Reply:
Thanks, updated